### PR TITLE
RELATED: RAIL-3989 Docs on how to access dashboard state externally

### PR DESCRIPTION
**Pull request by plavemar**
JIRA: RAIL-3989

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/958/ : SUCCESS in 1m 35s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/635/ : SUCCESS in 10s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/959/ : SUCCESS in 1m 30s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/636/ : SUCCESS in 8s

**Review by no23reason — APPROVED** [review]
LGTM

**Review by no23reason — APPROVED** [review]
LGTM

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/964/ : SUCCESS in 1m 48s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/641/ : SUCCESS in 8s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/965/ : SUCCESS in 1m 38s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/642/ : SUCCESS in 8s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/966/ : SUCCESS in 1m 27s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/643/ : SUCCESS in 5s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/971/ : SUCCESS in 1m 35s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/646/ : SUCCESS in 6s

File: docs/18_dashboard_component.md [hunk 1]
```diff
@@ -182,30 +182,24 @@ See all configuration options [here](https://github.com/gooddata/gooddata-ui-sdk
...
+* `getDashboardDispatch()` returns a `dispatch` object for the current dashboard.
+* `getOnChangeHandler()` returns a function that can be used as the `onStateChange` callback for the current dashboard.
+* `clearAccessor()` removes the accessor from the `store` accessor object.
+* `isAccessorInitialized()` returns `true` if the accessor is available for the current dashboard; otherwise, returns `false`.
+
+To access more than one dashboard `store` object, use the [DashboardStoreAccessorRepository](https://github.com/gooddata/gooddata-ui-sdk/blob/master/libs/sdk-ui-dashboard/src/model/store/storeAccessors/DashboardStoreAccessorRepository.ts) class. This class uses a map where the value is [DashboardStoreAccessor](https://github.com/gooddata/gooddata-ui-sdk/blob/master/libs/sdk-ui-dashboard/src/model/store/storeAccessors/DashboardStoreAccessor.ts) for a single dashboard and the key is the dashboard's `ObjRef`.
+
+If the default implementation of the `DashboardStoreAccessorRepository` class does not cover your specific scenario, you can use `DashboardStoreAccessor` to implement your own accessor repository:
```
Comment (plavemar) [inline docs/18_dashboard_component.md:196]: This is not related to the function list below. It is here to mention that users can implement the DashboardStoreAccessorRepository themselves if the DashboardStoreAccessorRepository we have provided is not suitable for them.

Comment (evgeniabogranskaya) [reply docs/18_dashboard_component.md]: Fixed

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/972/ : SUCCESS in 1m 14s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/647/ : SUCCESS in 9s

**Review by no23reason — APPROVED** [review]
LGTM

**Comment by yenkins** [comment]
### Sonar scan result
 - No issue found. Congratulations!

More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-ui-gate-PR486

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/973/ : SUCCESS in 1m 12s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-fossa-scanning-zuul-el7/194/ : SUCCESS in 43s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sonarqube-scan-zuul-gate/89/ : SUCCESS in 36s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-merge-compliance-check/284/ : SUCCESS in 18s
